nix-install.sh

```diff
@@ -106,9 +79,9 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        printf "\33[1minfo:\33[0m downloading installer \33[4m$_url\33[0m\n" 1>&2
```

Comment:
```suggestion
        printf "\33[1minfo:\33[0m downloading installer \33[4m%s\33[0m\n" "$_url" 1>&2
```

Shouldn't use variables in format srings